src/main.js

```diff
@@ -5,14 +5,13 @@ let path               = require('path');
 let debug              = require('debug')('server');
 let Promise            = require('promise');
 let AWS                = require('aws-sdk-promise');
-let raven              = require('raven');
 let exchanges          = require('./exchanges');
 let ScopeResolver      = require('./scoperesolver');
 let signaturevalidator = require('./signaturevalidator');
 let taskcluster        = require('taskcluster-client');
 let url                = require('url');
 let validate           = require('taskcluster-lib-validate');
-let loader             = require('taskcluster-lib-loader');
+let loader             = require('taskcluster-lib-loader')('process');
```

Comment:
There should not be `('process')` at the end here. Does it even run without stack tracing?